locales/sv-SE/src/studio.ts

```diff
@@ -649,11 +649,13 @@ export default removeUndefinedLocaleResources({
   /** Hotspot & Crop */
   'inputs.imagetool.title': 'Hotspot & Crop',
   /** Warnings displayed to developers when using the crop/hotspot tool on vector images, notifying them that crops/hotspot are not respected when serving the image in vector format. For the crop/hotspot to apply, images must be served in a raster format such as JPG or PNG, by appending eg `fm=jpg` to the image url, or calling `format('jpg')` if using `@sanity/image-url` */
-  'inputs.imagetool.vector-warning.developer-info': undefined, // 'The Asset Pipeline does not support hotspot and crop for vector formats. To enable hotspot & crop, output this image to any of the supported raster formats. For example: <code>fm=jpg</code> to the <ImageUrlDocumentationLink>image URL</ImageUrlDocumentationLink> or call <code>.format(\'png\')</code> with <ImageUrlPackageDocumentationLink>@sanity/image-url</ImageUrlPackageDocumentationLink>.'
+  'inputs.imagetool.vector-warning.developer-info':
+    "Asset Pipeline stöder inte hotspot och beskärning för vektorformat. För att aktivera hotspot & beskärning, exportera denna bild till något av de stödda rasterformaten. Till exempel: <code>fm=jpg</code> till <ImageUrlDocumentationLink>bildens URL</ImageUrlDocumentationLink> eller anropa <code>.format('png')</code> med <ImageUrlPackageDocumentationLink>@sanity/image-url</ImageUrlPackageDocumentationLink>.",
```

Comment:
"hotspot" could potentially be translated to "fokuspunkt", but not sure about that since we're in a "Sanity context" where hotspot maybe is more widely known, used and understood?